src/profile-logic/processed-profile-versioning.js

```diff
@@ -818,5 +818,42 @@ const _upgraders = {
       thread.stringArray = stringTable.serializeToArray();
     }
   },
+  [18]: profile => {
+    // When we added column numbers we forgot to update the func table.
+    // As a result, when we had a column number for an entry, the line number
+    // ended up in the `fileName` property, and the column number in the
+    // `lineNumber` property.
+    // We update the func table with right values of 'fileName', 'lineNumber' and 'columnNumber'.
+    for (const thread of profile.threads) {
+      const { funcTable, stringArray } = thread;
+      const stringTable = new UniqueStringArray(stringArray);
+      funcTable.columnNumber = [];
+      for (
+        let funcIndex = 0;
+        funcIndex < thread.funcTable.length;
+        funcIndex++
+      ) {
+        funcTable.columnNumber[funcIndex] = null;
+        if (funcTable.isJS[funcIndex]) {
+          const fileNameIndex = funcTable.fileName[funcIndex];
+          if (fileNameIndex !== null) {
+            const fileName = stringTable.getString(fileNameIndex);
+            const match = /^(.*):([0-9]+)$/.exec(fileName);
+            // If this regexp matches, this means that this is a lineNumber, and that the
+            // value in `lineNumber` is actually the column number.
```

Comment:
Hey @amelarbab, sorry for this nit, but because this comment is about what happens inside the if block, we prefer that it's inside the if block itself. So can you move the comment after the line `if (match)` please?